readme.md

```diff
@@ -135,23 +135,23 @@ $ npm -v
 
 * **Question:** What is `path` and where does it come from?
 
-* **Your Answer:**
+* **Your Answer: provides utilities for working with file and directory paths, it returns the current working directory
 
 ---
 
 - [ ] Install the [moment](https://www.npmjs.com/package/moment) package
 
 * **Question:** What command can you run to install this package?
 
-* **Your Answer:**
+* **Your Answer: npm install moment
 
 ---
 
 - [ ] On your own, use this package in the `index.js` file
 
 * **Question:** Do you need to use a `./` to require the package? Why or why not?
 
-* **Your Answer:**
+* **Your Answer: only if it is within your file system
```

Comment:
This isn't quite right. When you install `moment`, it's technically in your file system. It's installed inside of the `node_modules/` folder. Another way to put this might be that you only need to use the `./` if it is a file you have created.